File: src/app/src/main/java/com/onscreensync/tvapp/apirequests/ConfigApiRequest.java
package com.onscreensync.tvapp.apirequests;

import com.onscreensync.tvapp.apiresponses.ConfigApiResponse;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ConfigApiRequest {
    @GET("config.json")
    Call<ConfigApiResponse> loadConfig();
}

File: src/app/src/main/java/com/onscreensync/tvapp/apirequests/ConfigApiRequest.kt
package com.onscreensync.tvapp.apirequests

import com.onscreensync.tvapp.apiresponses.ConfigApiResponse
import retrofit2.Call
import retrofit2.http.GET

interface ConfigApiRequest {
    @GET("config.json")
    fun loadConfig(): Call<ConfigApiResponse>
}
